src/Components/Filtering/RangeFilterCard.js

```diff
@@ -0,0 +1,59 @@
+import React, { useRef } from 'react';
```

Comment:
Again, it would be helpful to rename the range filter files to better signify that one is a component for a singular filter and the other is a larger wrapper that handles all of the ranges